client/src/components/InputFormField/InputFormField.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Field} from 'formik';
+import { TextField } from '@mui/material';
+
+interface Props {
+  name: string;
+  type: string;
+  label: string;
+}
+
+const InputFormField: React.FC<Props> = (props) => {
```

Comment:
Better not use React.FC<Props>.
Instead use one of this:
`const name = (props: Props) => {}`
`const name = (props: Props): JSX.Element => {}`

Reference: [Remove React.FC](https://github.com/facebook/create-react-app/pull/8177).